packages/config/config/devices/0x0344/he-hls01.json

```
@@ -0,0 +1,449 @@
+				{
+					"label": "Momentary switch",
+					"value": 2
+				},
+				{
+					"label": "Reversed momentary switch",
+					"value": 3
+				}
+			]
+		},
+		"51": {
+			"label": "External Input: Click Control Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Click function disabled",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch",
+					"value": 1
+				},
+				{
+					"label": "Relay switches to ON state only",
+					"value": 2
+				},
+				{
+					"label": "Relay switches to OFF state only",
+					"value": 3
+				},
+				{
+					"label": "Timer On>Off mode",
+					"value": 4
+				},
+				{
+					"label": "Time Off>On mode",
+					"value": 5
+				}
+			]
+		},
+		"71": {
+			"label": "Relay Timer Mode Duration",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"141": {
+			"label": "Energy Consumption Meter Consecutive Report Interval",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 120,
+			"defaultValue": 10
+		},
+		"142": {
+			"label": "Energy Consumption Report Threshold",
+			"unit": "W",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 50,
+			"defaultValue": 25
+		},
+		"143": {
+			"label": "Temperature Sensor Consecutive Report Interval",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 120,
+			"defaultValue": 10
+		},
+		"144": {
+			"label": "Temperature Sensor Report Threshold",
+			"unit": "0.1 °C",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 2
+		},
+		"151": {
+			"label": "Device Failure Notifications",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"152": {
+			"label": "Temperature Rise Notification",
+			"description": ": Value of this Parameter should be lower than value of Parameter 153 (Overheat Protection & Notification).",
+			"unit": "°C",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 120,
+			"defaultValue": 1
+		},
+		"153": {
+			"label": "Overheat Protection & Notification",
+			"unit": "°C",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 120,
+			"defaultValue": 0
+		},
+		"154": {
+			"label": "Over-Current Notification",
+			"description": "The device will switch Off the load when the current reaches 16A.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"155": {
+			"label": "Over-Load Protection & Notification",
+			"description": "Allowable Range: 100-4000. 0 to disable function.",
```

```suggestion
			"description": "Allowable Range: 100-4000",
```
Please add a single `option` for `"Disable"`.

---

Same for the next parameters